src/EditorFeatures/Core/Host/IStreamingFindReferencesPresenter.cs

```diff
@@ -32,16 +32,18 @@ internal interface IStreamingFindUsagesPresenter
         /// It can also show messages about no references being found at the end of the search.
         /// If false, the presenter will not group by definitions, and will show the definition
         /// items in isolation.</param>
-        /// <param name="cancellationToken">External cancellation token controlling whether finding shoudl be canceled
-        /// or not.  This will be combined with a cancellation token owned by the <see cref="FindUsagesContext"/>.
-        /// Callers should consider <see cref="FindUsagesContext.CancellationToken"/> to be the source of truth for
-        /// cancellation from that point on.</param>
-        FindUsagesContext StartSearch(string title, bool supportsReferences, CancellationToken cancellationToken);
+        /// <returns>A cancellation token that will be triggered if the presenter thinks the search
+        /// should stop.  This can normally happen if the presenter view is closed, or recycled to
+        /// start a new search in it.  Callers should only use this if they intend to report results
+        /// asynchronously and thus relinquish their own control over cancellation from their own
+        /// surrounding context.  If the caller intends to populate the presenter synchronously,
+        /// then this cancellation token can be ignored.</returns>
+        (FindUsagesContext context, CancellationToken cancellationToken) StartSearch(string title, bool supportsReferences);
 
         /// <summary>
         /// Call this method to display the Containing Type, Containing Member, or Kind columns
         /// </summary>
-        FindUsagesContext StartSearchWithCustomColumns(string title, bool supportsReferences, bool includeContainingTypeAndMemberColumns, bool includeKindColumn, CancellationToken cancellationToken);
+        (FindUsagesContext context, CancellationToken cancellationToken) StartSearchWithCustomColumns(string title, bool supportsReferences, bool includeContainingTypeAndMemberColumns, bool includeKindColumn);
 
         /// <summary>
```

Comment:
Did you want to delete ClearAll since we're already here and breaking the interface anyways?

Reply:
@KirillOsenkov can you check if you guys use `ClearAll` at all in VS4mac for us?  thanks!